patterns/2-structured/innersource-license.md

```diff
@@ -32,6 +32,7 @@ At the time of sharing the source code, it can not be reliably predicted what th
 - Freedom over using the software leads to competition, and spread of ownership
 - There are legal contracts in place which cover the sharing of source code. These contracts are not standardized, so they create additional effort in negotiating and understanding for every project. The existing contracts may also not allow sharing source code in an open enough sense to support a true InnerSource approach.
 - Alternatively, there are no legal contracts in place but source code is shared informally. That might create uncertainty in cases where clarity about ownership and rights and obligations is needed.
+- **to be finalized**: wrong choice of license might inhibit transition to OSS
```

Comment:
@gruetter do you ave time to finalize this description?

If not we can also merge the PR as is, so that we at least get the Known Instance added. You could then add further descriptions in a separate PR.